modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-== Prerequisites
+CloudHub provides the ability to display notifications from your app in the Runtime Manager console.
+You can configure CloudHub to email these notifications as _custom alerts_ to specific people 
+so that they can fix the underlying problems.
 
-* Your application must be deployed to CloudHub. +
-xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
-* You Mule project must be mavenized. +
-** If you are using Studio 6.x and earlier, make sure you xref:6@studio::enabling-maven-support-for-a-studio-project.adoc[Mavenize your project in Studio].
-** All projects in Studio 7.x and later are mavenized by default.
 
+[NOTE]
+Custom alerts and notifications are available only for apps deployed to CloudHub workers.
+They are not available for apps deployed to on-premises servers.
+See xref:deployment-strategies.adoc[Deployment Options] for more information.
 
-== How to Use Notifications
 
-Notifications give you visibility into business-related events inside your application. For example, you can create notifications for when:
+== Prerequisites
 
-* Your application is unable to connect to a remote service.
-* An error that requires human intervention occurs, such as a problem with data mapping.
-* You want to create a summary of what occurred inside your application, such as the number of orders processed.
+* Deploy your application to CloudHub.
++
+See xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
+* For Anypoint Studio 6.x and earlier, enable Maven support for your Mule project.
++
+See xref:6@studio::enabling-maven-support-for-a-studio-project.adoc[Enabling Maven Support for a Studio Project].
++
+All projects in Studio 7.x and later are mavenized by default.
```

Comment:
Wow, I'll bet Apache is not very happy about its trade name becoming a verb....

Reply:
Seems to be a standard term. A quick google reveals a *-mavenize* option in RedHat.